sensorimotor/experiments/capacity/params/reasonable/2048.yaml

```diff
@@ -1,5 +1,7 @@
 n: 2048
```

Comment:
Yes, I thought we discussed having two sets of directories, one top level directory called `online` and one called `offline`, with param files in those directory structures. Then when we want to get rid of `offline`, we can just delete that directory (and remove the `isOnline` parameter).

Otherwise, how can you run the offline parameters to test currently?